core/src/components/header/header.ios.scss

```diff
@@ -9,13 +9,15 @@
 }
 
 @supports (backdrop-filter: blur(0)) {
-  .header-translucent-ios {
+  .header-background {
```

Comment:
Wouldn't this apply to MD too?

Reply:
It's in the header.ios.scss file, so it should be fine, right? Or do I still need `.header-translucent-ios .header-background`?

Reply:
I thought that scoped used to do that, but I'm seeing it work fine. I wonder why we specify the class + mode in all of the CSS files though instead of just using `.header-translucent` or `ion-header` then.